### PR TITLE
Upgrade postgres version

### PR DESCRIPTION
**Pull request by Menda**

### CODE REVIEW

File: service_builder/templates/docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 echo $(date -u) "- Environment: $environment"
 
 if [ "$environment" == "localhost" ]; then
+    bash tcp-port-wait.sh $DATABASE_HOST $DATABASE_PORT
```
Comment (bogdangi) [inline service_builder/templates/docker/docker-entrypoint.sh]: It installs every time `netcat` which is make the entrypoint is not stable. I think it should be part of docker container.

File: service_builder/templates/docker/Dockerfile [hunk 2]
```diff
@@ -10,6 +10,7 @@ ADD . /code
 ARG environment=production
 ENV environment=${environment}
 RUN echo "Environment: $environment"
+RUN if [ "$environment" = "localhost" ] ; then apt update && apt install -y netcat; fi
```
Comment (bogdangi) [inline service_builder/templates/docker/Dockerfile]: We have to remove environment dependency during the build.
The image should be always the same as production one.
There should be no dependency on environment.

**Comment by Menda** [comment]
@bogdangi Reading this comment then I don't know really well what to do:

```

**Comment by bogdangi** [comment]
Just remove environment variable in docker file.

**Comment by Menda** [comment]
I can't. The entrypoint needs to know in which environment are we executing the app

**Comment by bogdangi** [comment]
Think about that if your build depends on some state it can be that your docker container behaves differently. And we will have in the future "It works on my computer". because for you computer it builds in different way that it builds for production/staging/demo and so on.

File: service_builder/templates/docker/Dockerfile [hunk 3]
```diff
@@ -4,6 +4,8 @@ WORKDIR /code
 
 COPY ./requirements/base.txt requirements/base.txt
 RUN pip install -r requirements/base.txt
+# Install tcp-port-wait requirements
+RUN apt update && apt install -y netcat
```
Comment (bogdangi) [inline service_builder/templates/docker/Dockerfile]: Could you move it before pip install?

**Pull request merged**